_inc/connect-button.js

```diff
@@ -50,9 +50,12 @@ jQuery( document ).ready( function( $ ) {
 			jetpackConnectButton.isRegistering = true;
 			tosText.hide();
 			connectButton
-				.text( jpConnect.buttonTextRegistering )
-				.attr( 'disabled', true )
-				.blur();
+				.hide()
+				.after(
+					'<span class="jp-connect-full__button-container-loading">' +
```

Comment:
What do you think of having this `span` in the markup from the very beginning and just showing it here?
Or else - constructing the element like:
```js
var loadingText = $( 'span' );
loadingText.addClass( 'jp-connect-full__button-container-loading' );
loadingText.text( jpConnect.buttonTextRegistering );
```
I think it would be safer and is actually recommended by [VIP](https://wpvip.com/documentation/vip-go/vip-code-review/javascript-security-best-practices/#overview).

Reply:
I like that 👍

Reply:
Great Idea will update it to that! Thanks for the suggestion